server/src/main/java/org/elasticsearch/TransportVersions.java

```diff
@@ -327,7 +343,7 @@ public static NavigableMap<Integer, TransportVersion> getAllVersionIds(Class<?>
             }
         }
 
-        return Collections.unmodifiableNavigableMap(builder);
+        return builder;
```

Comment:
This is no longer a builder-type object now

Reply:
Renamed